src/ContractEditor/plugins/clausePlugin.js

```diff
@@ -2,10 +2,6 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Icon } from 'semantic-ui-react';
 import styled from 'styled-components';
-import { Template, Clause } from '@accordproject/cicero-core';
-import BadParse from '../../SlateCommands/BadParse';
```

Comment:
Should the `BadParse` `GoodParse` `SetNodeData` be removed as well? Or are those called from the editor instead?

Reply:
as far as I can see, they are not being used anywhere else. I'll remove those files, thanks! We'll be implementing new functions to add annotations once Slate pushes a fix to that bug that Dan was experiencing.